pkg/eval/glob.go

```diff
@@ -75,6 +89,24 @@ func (gp GlobPattern) Index(k interface{}) (interface{}, error) {
 		gp.Segments[len(gp.Segments)-1] = glob.Wild{
 			lastSeg.Type, true, lastSeg.Matchers,
 		}
+	case modifier == "d" || modifier == "r" || strings.HasPrefix(modifier, "type:"):
+		// This handles cases such as `**x*[r][d]`.
```

Comment:
Change this comment to use the longhand form, or just remove it as it's quite self-evident what this branch does.